tools/building.py

```diff
@@ -1145,8 +1146,11 @@ def emit_wasm_source_map(wasm_file, map_file, final_wasm):
   # source file paths must be relative to the location of the map (which is
   # emitted alongside the wasm)
   base_path = os.path.dirname(os.path.abspath(final_wasm))
-  sourcemap_cmd = [sys.executable, '-E', path_from_root('tools/wasm-sourcemap.py'),
-                   wasm_file,
+
+  # TODO(sbc): Rename wasm-sourcemap so it can be imported directly without
+  # importlib.
```

Comment:
Why can't we directly import it unless we don't rename it?

Reply:
Becuase of the hyphen in the name, you can't write `import foo-bar` directly in python since it would interpret the `-` as a minus sign.